File: app/src/main/java/com/tylerhats/proxitap/audio/TestWebRtc.java
package com.tylerhats.proxitap.audio;

import android.content.Context;
import org.webrtc.audio.JavaAudioDeviceModule;

public class TestWebRtc {
    public static void printMethods() {
        try {
            Class<?> clazz = Class.forName("org.webrtc.audio.JavaAudioDeviceModule$Builder");
            for (java.lang.reflect.Method m : clazz.getMethods()) {
                System.out.println("METHOD_FOUND: " + m.getName() + " -> " + m.getParameterTypes().length + " args");
            }
        } catch (Exception e) {}
    }
}

File: app/src/main/java/com/tylerhats/proxitap/audio/TestWebRtc.kt
package com.tylerhats.proxitap.audio

import android.content.Context
import org.webrtc.audio.JavaAudioDeviceModule

fun testADM(context: Context) {
    val builder = JavaAudioDeviceModule.builder(context)
    val methods = builder::class.java.methods
    for (m in methods) {
        println("BUILDER_METHOD: ${m.name}")
    }
}
